libs/designsystem/src/lib/components/card/card.component.scss

```diff
@@ -31,6 +31,10 @@
   &.flat {
     box-shadow: none;
   }
+
+  &.card-is-clickable {
+    cursor: pointer;
+  }
```

Comment:
This could be replaced with 
```suggestion
  &[role="button"] {
    cursor: pointer;
  }
```
if you agree with my suggestions to the directive (below)

Reply:
And I just talked with @henrikvoetmand. We would also like an outline-offset to the focus ring of 2px (`outline-offset: 2px;`). Remember to use a size variable here.